packages/apps/crm/src/screens/event/EventFormScreen.js

```diff
@@ -0,0 +1,181 @@
+/*
+ * Axelor Business Solutions
+ *
+ * Copyright (C) 2023 Axelor (<http://axelor.com>).
+ *
+ * This program is free software: you can redistribute it and/or  modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React, {useCallback, useMemo} from 'react';
+import {FormView, useSelector} from '@axelor/aos-mobile-core';
+import {EventType} from '../../types';
+import {createEvent, updateEvent} from '../../features/eventSlice';
+
+const MODELS = {
+  lead: 'com.axelor.apps.crm.db.Lead',
+  partner: 'com.axelor.apps.base.db.Partner',
+};
+
+const EventFormScreen = ({navigation, route}) => {
+  const event = route?.params?.event;
+  const lead = route?.params?.lead;
+  const prospect = route?.params?.prospect;
+  const client = route?.params?.client;
+  const contact = route?.params?.contact;
+
+  const {user} = useSelector(state => state.user);
+
+  const defaultValue = useMemo(() => {
+    const _defaultStartDate = new Date().toISOString();
+    const _defaultEndDate = new Date(
+      new Date().setHours(new Date().getHours() + 1),
+    ).toISOString();
```

Comment:
This can be simplify. toISOString can be done directly in default construction
Are you sure that the last new Date is required on end date ? I think the following should work 🤔

```
    const _defaultStartDate = new Date();
    const _defaultEndDate = new Date().setHours(_defaultStartDate.getHours() + 1);
```